moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame.cpp

```diff
@@ -281,11 +281,13 @@ void MotionPlanningFrame::fillStateSelectionOptions()
     ui_->start_state_combo_box->addItem(QString("<random>"));
     ui_->start_state_combo_box->addItem(QString("<current>"));
     ui_->start_state_combo_box->addItem(QString("<same as goal>"));
+    ui_->start_state_combo_box->addItem(QString("<previous>"));
```

Comment:
Do we need this for the start state? Can you think of any use case?
If so, please update the start state in `MotionPlanningFrame::onFinishedExecution` as well (see below).

Reply:
I suppose a use case to find an alternative path for the latest robot motion.
So, I will change as you mentioned.

Reply:
I figured that if `<random>` is an option for the start state, this should be as well. I also couldn't think of a use case outside of finding alternative paths.